src/layouts/AlertLayout.tsx

```diff
@@ -2,15 +2,42 @@ import { LabelProps, RendererProps } from "@jsonforms/core"
 import { Alert } from "antd"
 import { AlertLayoutOptions } from "../ui-schema"
 import { withJsonFormsLabelProps } from "@jsonforms/react"
+import ReactMarkdown from "react-markdown"
 
 export function AlertLayout({ text, uischema }: LabelProps & RendererProps) {
-  const options = uischema.options as AlertLayoutOptions
+  const options = (uischema?.options ?? { style: {} }) as AlertLayoutOptions
   return (
     <Alert
-      style={{ marginBottom: "24px" }}
-      type={options?.type}
-      message={text}
-      showIcon
```

Comment:
This is a breaking change--you could add it to `AlertLayoutOptions`'s pick of `AlertProps` if you want it to be configurable